indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -3648,25 +3704,33 @@ private void updateCurrentOffsets() throws InterruptedException, ExecutionExcept
 
   protected Map<PartitionIdType, SequenceOffsetType> getHighestCurrentOffsets()
```

Comment:
Please add a javadoc to this method and maybe explain how this change affects the idle behaviour.